src/pages/Payment/Payment.jsx

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import Button from '../../components/Button/Button';
+import Ordersinfo from './components/Ordersinfo';
 import './Payment.scss';
 
 const Payment = () => {
-  return <div>payment</div>;
+  return (
+    <div className="payment">
+      <div className="paymentArea">
+        <h1 className="paymentTitle">예매정보</h1>
+        <Ordersinfo
+          title="뮤지컬 고양이"
+          imgUrl="/images/visual_01.png"
+          time="150분"
+          date="2023년 5월 5일 오후 2시"
+          location="수원아트센터"
+          seatNumber="R열 35번"
+          totalPrice="70,000"
+        />
+        <div className="tossArea">toss API</div>
+        <Button btnText="결제완료" />
```

Comment:
Button 공용 컴포넌트 리뷰 때 놓쳤는데, 버튼 텍스트는 props로 받는 것이 아닌 children 형태로 받는 게 일반적인 버튼의 사용 형태에 맞을 것 같네요.
```suggestion
        <Button>결제완료</Button>
```
이 형태로 작성할 수 있도록 지영님에게 전달해 주세요-!

Reply:
버튼 공용 컴포넌트 부분은 지영님이 따로 PR올려주시고 버튼부분만 수정했습니다!